migrations/20190212154756-create-user-token.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("UserTokens", {
```

Comment:
@abayo-luc we agreed that tables names should start with lowercase, check the names of others tables